convert/gitlab/convert.go

```diff
@@ -422,9 +432,12 @@ func convertImageAndPullPolicy(image *gitlab.Image) (string, string) {
 
 			pullPolicy = pullPolicyMapping[image.PullPolicy[0]]
 		}
+		if len(image.Entrypoint) > 0 {
+			entryPoint = image.Entrypoint[0]
```

Comment:
Can we take all elements of the `image.Entrypoint` list and convert them to a string? Then we can support `entrypoint` with multiple args.

Reply:
When invoking a command in Go / Posix the interface looks something like this:

```Go
Exec(command string, args []string)
```

so I recommend would be to do something like this:

```Go
if len(image.Entrypoint) > 0 {
  step.Entrypoint = image.Entrypoint[0]
  if len(image.Entrypoint > 1 {
    step.Args = image.Entrypoint[1:]
  }
}
```

The reason that combining an entrypoint with multiple values into a string won't work is because that would create a bash / shell string. In this case, we are not invoking processes using bash, we are invoking processes using the posix system call which requires structured input.

Reply:
done.